core/ledger/statemgmt/trie/trie_node.go

```diff
@@ -134,7 +134,7 @@ func (trieNode *trieNode) marshal() ([]byte, error) {
 	buffer := proto.NewBuffer([]byte{})
 
 	// write value marker explicitly because rocksdb apis convertes a nil into an empty array and protobuf does it other-way around
-	var valueMarker uint64 = 0
+	var valueMarker uint64 // default value is 0
```

Comment:
Now this is a lint warning that I would have ignored.  The assignment is more clear than the comment.